packages/cli/src/commands/devPlugin.ts

```diff
@@ -27,20 +28,21 @@ export const devPlugin: DevPluginCommandHandler = async ({ type, name }) => {
   let dirname;
   try {
     dirname = path.join(process.cwd(), name);
-    if (fse.existsSync(dirname)) {
+    const isDirExist = await promisify(fse.exists)(dirname);
```

Comment:
`fse.exists` doesn't need a promisify, it returns a Promise

Reply:
no, it has callback function